style.css

```diff
@@ -89,3 +89,253 @@
 .social li {
   margin: 10px;
 }
+
+/* work section */
+
+.work-section {
+  display: flex;
```

Comment:
- [ ] As a project requirement, please 🙏🏽 use display: grid for the container of the works section.

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.